packages/admin-ui/src/Radio/Radio.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import * as RadioGroupPrimitive from "@radix-ui/react-radio-group";
+import { cn, makeDecoratable } from "~/utils";
+
+/**
+ * RadioItem
+ */
+type RadioProps = React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Item> & {
```

Comment:
I noticed you're using `type` in some of these. 

Which is fine, but I believe we're generally leaning more towards using `interface` when possible. So in this case, we could do:

```ts
interface RadioProps extends React.ComponentPropsWithoutRef<typeof RadioGroupPrimitive.Item> {
   label: string | React.ReactNode;
}
```

Just mentioning it, it's a nitpick. Bu feel free to leave it if you want.

Reply:
Thank you, I have switched to `interface` in this component. I will implement the same change in other components as well.